apps/client/app/globals.css

```diff
@@ -0,0 +1,33 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+:root {
```

Comment:
I realized you probably don't need these variables here at all 😄 you can declared them only in tailwind theme and use them here for `body` with tailwind classes this way:
```css
body {
  @apply text-text-color bg-background-color;
}
```

Reply:
...and `@apply` keyword is a general way to pass tailwind classes in `.css` files 👌